osu.Game/Screens/Play/Player.cs

```diff
@@ -660,10 +671,10 @@ protected void SetGameplayStartTime(double time)
         /// <remarks>This can be called from a child screen in order to trigger the restart process.</remarks>
         /// </summary>
         /// <param name="quickRestart">Whether a quick restart was requested (skipping intro etc.).</param>
-        public void Restart(bool quickRestart = false)
+        public bool Restart(bool quickRestart = false)
```

Comment:
Sorry for a nitpick: 
also need to add `/// <returns>` doc like for `PerformExit`?